filterconfig/filterconfig.go

```diff
@@ -96,18 +96,18 @@ type TokenUsageMetadata struct {
 
 // VersionedAPISchema corresponds to LLMAPISchema in api/v1alpha1/api.go.
 type VersionedAPISchema struct {
-	// Schema is the API schema.
-	Schema APISchema `yaml:"schema"`
+	// Name is the name of the API schema.
+	Name APISchemaName `yaml:"name"`
 	// Version is the version of the API schema. Optional.
 	Version string `yaml:"version,omitempty"`
 }
 
-// APISchema corresponds to APISchema in api/v1alpha1/api.go.
-type APISchema string
+// APISchemaName corresponds to APISchemaName in api/v1alpha1/api.go.
+type APISchemaName string
```

Comment:
Given that the values are provider names, wouldn't APISchemaProviderName be more suitable than APISchemaName?

Reply:
I dont have a big preference, open for discussion on this